backend/fleet_management/serializers.py

```diff
@@ -41,4 +46,24 @@ class Meta:
             'id',
             'driver', 'car', 'passengers',
             'date', 'start_mileage', 'end_mileage', 'description',
+            'start_location', 'end_location',
         )
+
+    def create(self, validated_data):
+        # tracks_data = validated_data.pop('tracks')
```

Comment:
Why is this line commented out?

Reply:
Good spot :D I copied it from tutorial when I was trying to figure out how process nested data.